base/applications/taskmgr/lang/it-IT.rc

```diff
@@ -313,21 +313,21 @@ END
 
 STRINGTABLE
 BEGIN
-    IDS_APP_TITLE "Task Manager"
-    IDC_TASKMGR "Task Manager"
+    IDS_APP_TITLE "Task manager"
+    IDC_TASKMGR "Task manager"
```

Comment:
Can you please attach a picture of 2k3 italian taskmgr for proof that they do use uncapitalized "manager" here?

Reply:
Just checked it, it has the 'M' capitalized. Gonna fix it.

Reply:
resolved.